src/App.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { TextareaHTMLAttributes, useState } from 'react';
+import './App.css';
+import './scripts/pokemondbScraper'
+
+import NavBar from './components/NavBar'
+import Clickable from './components/Clickable'
+import EssayForm from './components/EssayForm'
+import TabPanel from './components/TabPanel'
+import PokemonDisplay from './components/PokemonDisplay'
+
+import Button from '@material-ui/core/Button'
+import Tabs from '@material-ui/core/Tabs'
+import Tab from '@material-ui/core/Tab'
+import Box from '@material-ui/core/Box'
+import SearchBar, { GetSearchBarText } from './components/SearchBar';
+
+
+const App: React.FC = () => {
+    const [isPrimary, setIsPrimary] = useState(false);
+    const [currentTab, setCurrentTab] = useState(0);
+    const [searchText, setSearchText] = useState("");
+
+    function handleStatusChange(status: boolean) {
+      setIsPrimary(status);
+    }
+    const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
+      setCurrentTab(newValue);
+    };
+
+    const something=(event: any)=> {
+      if (event.keyCode === 13) {
+          setSearchText(GetSearchBarText());
+      }
+    }
+      return (
+        <div className="App" 
+            onKeyDown={(e) => {
```

Comment:
Probably unnecessary delegate function wrapper, try updating to just assigning onKeyDown to your handler function

Reply:
Did as suggested.